core/app/models/spree/tax/output/order.rb

```diff
@@ -0,0 +1,10 @@
+module Spree
+  module Tax
+    module Output
+      class Order
+        include ActiveModel::Model
+        attr_accessor :id, :line_items, :shipments
```

Comment:
We allow that only as legacy pre-spree-2.2 data. The current `Tax::OrderAdjuster` and `OrderUpdater` don't support it so neither should this.